git-config/src/lib.rs

```diff
@@ -30,6 +30,15 @@ impl From<Span> for Range<usize> {
     }
 }
 
+impl From<Range<usize>> for Span {
```

Comment:
Since `dangerous` uses a `Range<usize>` I might just use this instead of an own version of `..=`. Probably I had a reason to use an inclusive range in the first place, but it's not entirely clear to me right now what that was 🤦‍♂️.